Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java

```diff
@@ -57,50 +68,19 @@ public RandomCutForestState convert(URL url, Precision precision) throws IOExcep
     }
 
     public RandomCutForestState convert(V1SerializedRandomCutForest serializedForest, Precision precision) {
-        RandomCutForestState state = new RandomCutForestState();
-        state.setNumberOfTrees(serializedForest.getNumberOfTrees());
-        state.setDimensions(serializedForest.getDimensions());
-        state.setTimeDecay(serializedForest.getLambda());
-        state.setSampleSize(serializedForest.getSampleSize());
-        state.setShingleSize(1);
-        state.setCenterOfMassEnabled(serializedForest.isCenterOfMassEnabled());
-        state.setOutputAfter(serializedForest.getOutputAfter());
-        state.setStoreSequenceIndexesEnabled(serializedForest.isStoreSequenceIndexesEnabled());
-        state.setTotalUpdates(serializedForest.getExecutor().getExecutor().getTotalUpdates());
-        state.setCompact(true);
-        state.setInternalShinglingEnabled(false);
-        state.setBoundingBoxCacheFraction(1.0);
-        state.setSaveSamplerStateEnabled(true);
-        state.setSaveTreeStateEnabled(false);
-        state.setSaveCoordinatorStateEnabled(true);
-        state.setPrecision(precision.name());
-        state.setCompressed(false);
-        state.setPartialTreeState(false);
-
-        ExecutionContext executionContext = new ExecutionContext();
-        executionContext.setParallelExecutionEnabled(serializedForest.isParallelExecutionEnabled());
-        executionContext.setThreadPoolSize(serializedForest.getThreadPoolSize());
-        state.setExecutionContext(executionContext);
-
-        SamplerConverter samplerConverter = new SamplerConverter(state.getDimensions(),
-                state.getNumberOfTrees() * state.getSampleSize() + 1, precision);
-
-        Arrays.stream(serializedForest.getExecutor().getExecutor().getTreeUpdaters())
-                .map(V1SerializedRandomCutForest.TreeUpdater::getSampler).forEach(samplerConverter::addSampler);
-
-        state.setPointStoreState(samplerConverter.getPointStoreState(precision));
-        state.setCompactSamplerStates(samplerConverter.compactSamplerStates);
-
-        return state;
+        ArrayList<V1SerializedRandomCutForest> newList = new ArrayList<>();
```

Comment:
suggestion. for single-item list, there is Collections.singletonList or Arrays.asList.

Reply:
If you suggest the syntax to write, I am happy to change.

Reply:
yes, it's a suggestion for constructing a list with a known number of elements.

Reply:
the suggested code is
```
return convert(Arrays.asList(serializedForest), serializedForest.getNumberOfTrees(), precision);
```

Reply:
Done. Java IDE suggested "return convert(Collections.singletonList(serializedForest), serializedForest.getNumberOfTrees(), precision);"